### ISSUE TITLE
DISCUSSION: Possible work areas for the 1.0 release

### ISSUE THREAD

**Issue by dimaip** [neos/neos-ui#1043]
Decision to go into the following direction:

BUDGET (as of 16. October 2016)

* spent: 206 h / 26 days
* 22 days to go
  *  Polish the styling. Take into account both the old UI and the various drafts that we had, e.g. awesome suggestions bug Christopher Ono. As Sebastian says, Flo can help with it. -- 3-4 Days (Flo)
  * 4 PT Sebastian - for 1&1 "specials" / extensibility examples
  * 2 PT Performance #887
  * 2 PT Keyboard Handling -> Max
  * 1 PT speed up loading of UI -> Wilhelm
  * 1 PT Max/Sebastian -- Update to React 16
  * 4 PT try out Structured Editing / Create Concept -> Sebastian #1146
  * 2 PT Drag/Drop in content canvas #1147
  * Flo check End-user issues
  * 5 PT extra time/bugfixes as they come in / communication / post-1.0 bugfixes

### Done

- Punkt.de Project testing (ongoing)
- Better unplanned extensibility by making component registries support array-like structures. #1044  -> Dmitri

### Won't do before 1.0

- Feedback Handlers -> "reload document" should be moved client side -> Wilhelm
- #1034 Integrate all the editors in the creating dialog
- Migrate to GraphQL and better design the app's structure
- Migrate Google Analytics views

**Comment by dimaip** [comment]
> Integrate all the editors in the creating dialog
actually I'll do it during the Vienna sprint on personal time, since it's kind of my personal itch ~Dmitri*)

> Migrate Google Analytics views
This will be done by porting data views, that's exactly what I'm doing right now.

**Comment by dfeyer** [comment]
As said in Slack: Some many ideas now that the UI is a thing again :wink: but please for the v1, focus on what we have, we need to have a better overview of the futur UI kit, before starting to build stuff here and here

**Comment by dimaip** [comment]
@dfeyer I can assure you that bug fixing and implementing missing features is our strict priority, but the "problem" is that we don't have enough bugs left, so we can afford to give some efforts to do technical implementation of some directions of development we all agree on, like structured editing :smile:

Once UX guild gets together and comes up with a unified concept of visual UX improvements, we will be able to easily apply them on top of the technical foundations we are laying down now.
